ui/src/cloud/apis/demodata.ts

```diff
@@ -75,17 +74,30 @@ export const deleteDemoDataBucketMembership = async (
   }
 }
 
-export const getDemoDataBucketsFromAll = async (): Promise<Bucket[]> => {
+export const fetchDemoDataBuckets = async (): Promise<Bucket[]> => {
```

Comment:
does this return a `Promise<Bucket[]>` or just `Bucket[]`?

Reply:
It's returning a Promise because it is an async function.